client/components/marketing-survey/gsuite-cancel-purchase-dialog/gsuite-cancellation-features.jsx

```diff
@@ -17,29 +18,71 @@ class GSuiteCancellationFeatures extends Component {
 		this.props.recordTracksEvent( 'calypso_purchases_gsuite_remove_purchase_learn_more_click' );
 	};
 
+	getAccessMessage = () => {
+		const { googleSubscriptionStatus, purchase, translate } = this.props;
+		const { meta: domainName, productSlug } = purchase;
+
+		const googleMailService = getGoogleMailServiceFamily( productSlug );
+
+		if ( [ 'suspended', '' ].includes( googleSubscriptionStatus ) ) {
+			return translate(
+				'If you cancel your subscription for %(domainName)s now, {{strong}}you will lose access to all of ' +
+					'your %(googleMailService)s features immediately{{/strong}}, and you will ' +
+					'need to purchase a new subscription with Google if you wish to regain access to the features.',
```

Comment:
I think we could simplify the last part of the sentence with:

```suggestion
					'need to purchase a new subscription with Google if you wish to regain access to them.',
```